src/app/functions/server/logging.ts

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { createLogger, transports } from "winston";
+import { LoggingWinston } from "@google-cloud/logging-winston";
+
+const loggingWinston = new LoggingWinston({
+  labels: {
+    name: "monitor-stats",
+    version: "0.1.0",
+  },
+});
+
+export const logger = createLogger({
+  level: "info",
+  transports: [new transports.Console()],
```

Comment:
I think `test` or `development`, so you can see them in local development?

Reply:
I don't have a strong opinion on showing them in `development` - it could get noisy, but it could also be useful when debugging. But in tests I think they're very likely to be noise indeed.